cloudsmith/data_source_repository_privileges.go

```diff
@@ -82,7 +82,10 @@ func dataSourceRepositoryPrivilegesRead(d *schema.ResourceData, m interface{}) e
 	repository := d.Get("repository").(string)
 
 	req := pc.APIClient.ReposApi.ReposPrivilegesList(pc.Auth, organization, repository)
-
+	// TODO: add a proper loop here to ensure we always get all privs,
+	// regardless of how many are configured.
+	req = req.Page(1)
+	req = req.PageSize(1000)
```

Comment:
question: I'm not super familiar with go, is it standard to reassign to a var like is being done with req? Do Page and PageSize return the req object?

Reply:
Not a GO expert either but this was here before, I think we removed it during one of the releases but it turned out to be breaking stuff.